<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>精武堂</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/new/prefix1/jwt.png</normalon>
   </iconset>
  </property>
  <widget class="QLabel" name="welcome_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎您，新人【233】（3级）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="jwtresource.qrc">:/new/prefix1/jwt.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>212</y>
     <width>421</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="myState_verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="myStateAndMyFriends_horizontalLayout">
        <item>
         <widget class="QLabel" name="myState_label">
          <property name="text">
           <string>我的状态 |</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refresh_pushButton">
          <property name="text">
           <string>刷新状态</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="myFriends_pushButton">
          <property name="text">
           <string>我的好友</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="trainingRoomStateAndStartOrStop_horizontalLayout">
        <item>
         <widget class="QLabel" name="trainingRoomState_label">
          <property name="text">
           <string>练功房：未修练.....</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="trainingStartOrStop_pushButton">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="EXPAndUpgrade_horizontalLayout">
        <item>
         <widget class="QLabel" name="EXP_label">
          <property name="text">
           <string>经验：200/252</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="upgrade_pushButton">
          <property name="text">
           <string>升级</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="qiAndAndAddQi_horizontalLayout">
        <item>
         <widget class="QLabel" name="qi_label">
          <property name="text">
           <string>气血：5/30</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="betweenMyStateAndFiight_verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="fightSlogan_label">
          <property name="text">
           <string>【比武】</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fight1_horizontalLayout">
        <item>
         <widget class="QLabel" name="fight1_label">
          <property name="text">
           <string>1.新人1（1级））</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fight1_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="fight1_pushButton">
          <property name="text">
           <string>比武</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fight2_horizontalLayout">
        <item>
         <widget class="QLabel" name="fight2_label">
          <property name="text">
           <string>2.新人1（1级））</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fight2_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="fight2_pushButton">
          <property name="text">
           <string>比武</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fight3_horizontalLayout">
        <item>
         <widget class="QLabel" name="fight3_label">
          <property name="text">
           <string>3.新人1（1级））</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fight3_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="fight3_pushButton">
          <property name="text">
           <string>比武</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fight4_horizontalLayout">
        <item>
         <widget class="QLabel" name="fight4_label">
          <property name="text">
           <string>4.新人1（1级））</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fight4_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="fight4_pushButton">
          <property name="text">
           <string>比武</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fight5_horizontalLayout">
        <item>
         <widget class="QLabel" name="fight5_label">
          <property name="text">
           <string>5.新人1（1级））</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fight5_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="fight5_pushButton">
          <property name="text">
           <string>比武</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="betweenFightAndFunction_verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="function_gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="attribute_pushButton">
        <property name="text">
         <string>属性</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="shop_pushButton">
        <property name="text">
         <string>商店</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="charge_pushButton">
        <property name="text">
         <string>充值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bag_pushButton">
        <property name="text">
         <string>行囊</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="jwtresource.qrc"/>
 </resources>
 <connections/>
</ui>
